### DISCUSSION TITLE
could not find artifact: `SimpleStorage

### DISCUSSION THREAD

**Discussion by Monika0** [Cyfrin/foundry-full-course-cu#1680] [Q&A] [answered]
Hi there, i have an issue with forge create command, getting: could not find artifact: `SimpleStorage.
I've doublechecked namings and it looks okay to me. I've even clone the repo from the main course and facing the same issue. Any ideas what coud be wrong? 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/12182927/bc30b699-38da-4031-b609-f1dfe0953a14)

**Comment by EngrPips** [comment]
Hello @Monika0, Please re-run the command you are trying to run and then show a screenshot of the output from Foundry. Total output, please.

**Reply by Monika0** [reply]
Hi @EngrPips , here it is:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/12182927/0cfa9938-adbc-45ed-98d1-598bbcda0f01)

**Reply by EngrPips** [reply]
Hello @Monika0, So i tried to create a `SimpleStorage` contract with `forge create` command and it was successful. So this seem like an issue on your end. Have you built the contract with `Forge build` before trying to use `forge create`?

<img width="1440" alt="Screenshot 2024-05-04 at 18 15 27" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/5b15032c-8d4c-4aa2-92d7-41bf71716c83">

**Reply by Monika0** [reply, marked as answer]
Hi @EngrPips , yes im using forge build. I just realized that i specified the create command a bit incorrectly since im providing only the name of contract but not the contract path. so should be like:
 forge create src/SimpleStorage.sol:SimpleStorage --rpc-url http://127.0.0.1:7545 --interactive
Now im not getting that error at least :)

**Reply by Monika0** [reply]
And i provided a FK and it works perfectly. 
Thanks for your time anw

**Reply by EngrPips** [reply]
Amazing, I am so glad to hear that you fix it @Monika0.
